paper.md

```diff
@@ -1,53 +1,98 @@
+FHD [@fhd] and MIR [@mir] visibility save files. Additionally, pyuvdata supports reading
+and writing measurement set, CalFITS [@calfits], and CalH5 (Hazelton et al., _in prep_)
+calibration tables; and reading of FHD calibration tables. pyuvdata also provides
+interfaces for and handling of models of antenna primary beams, including reading and
+writing of BeamFITS [@beamfits] and MWA-formatted beam models, as well as for flags
+tables.
 
+# Statement of Need
 There are several standard formats for astronomical interferometric data, but
-converting between them in a stable and repeatable way has historically been
-very challenging. This is partly because of subtle assumptions in the
-implementations of the formats and the complexity of the mathematical
-relationships between the different formats (e.g. drift mode vs phased data)
-and partly because data analysis for individual telescopes
-typically used just one of the standards along with the associated analysis
-code. New low frequency instruments (e.g. MWA (http://www.mwatelescope.org/),
-PAPER (http://eor.berkeley.edu/), HERA (http://reionization.org/)),
-have required custom analysis and simulation software that rely on a range of
-different file formats. pyuvdata was designed to facilitate interoperability
-between these instruments and codes by providing high quality, well documented
-conversion routines as well as an interface to interact with interferometric
-data and simulations directly in python.
+converting between them in a stable, repeatable way has historically been
+challenging.  This is partially due to conflicting assumptions and standards, giving
+rise to significant (though sometimes subtle) differences between formats.
+Interfacing with different data formats -- like one does when they convert from one
+format to another -- thus requires careful accounting for the complex mathematical
+relationships between both data and metadata to ensure proper data fidelity. This is
+required both for leveraging existing community-favored tools that typically built
+to interface with a specific data format, as well as analyses requiring bespoke tools
+for specialized types of analyses and simulations leveraging data in a variety of
+formats.
 
-pyuvdata currently supports reading and writing uvfits [@uvfits] and
-miriad [@miriad] files and reading FHD [@fhd] visibility save files.
+pyuvdata has been designed to facilitate interoperability between different instruments
+and codes by providing high quality, well documented conversion routines as well as an
+interface to interact with interferometric data and simulations directly in Python.
+Originally motivated to support new low frequency instruments (e.g. MWA
+(http://www.mwatelescope.org/), PAPER (http://eor.berkeley.edu/), HERA
+(http://reionization.org/)), the capabilities of pyuvdata have been steadily expanded
+to support handling of data from several telescopes, ranging from meter to submillimeter
+wavelengths.
```

Comment:
Is it worth listing some here?

Reply:
Sure -- added in 246deac2f0f5c39ca465b7db483b6735af6af000.